testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/miniapplist/MiniAppListFragment.kt

```diff
@@ -71,26 +80,86 @@ class MiniAppListFragment : BaseFragment(), MiniAppList {
                     Toast.makeText(context, it, Toast.LENGTH_LONG).show()
                 else
                     displayMiniAppList(list)
-                swipeRefreshLayout.isRefreshing = false
+                    downloadedList = list
+                    swipeRefreshLayout.isRefreshing = false
             })
         }
 
-        swipeRefreshLayout.setOnRefreshListener { executeLoadingList() }
+        swipeRefreshLayout.setOnRefreshListener {
+            executeLoadingList()
+            resetSearchBox()
+        }
     }
 
     private fun displayMiniAppList(list: List<MiniAppInfo>) {
         miniAppListAdapter.addListWithSection(list)
+        updateEmptyView(list)
     }
 
     private fun executeLoadingList() {
         viewModel.getMiniAppList()
     }
 
     override fun onMiniAppItemClick(miniAppInfo: MiniAppInfo) {
-        raceExecutor.run { MiniAppDisplayActivity.start(context!!, miniAppInfo) }
+        raceExecutor.run {
+            context?.let { MiniAppDisplayActivity.start(it, miniAppInfo) }
+            resetSearchBox()
```

Comment:
This looks a little strange because the mini app list resets before the `MiniAppDisplayActivity` has started. Could you instead move this `resetSearchBox()` call to `onStart`?

Reply:
`resetSearchBox()` requires `searchView` property to be initialized. 
Basically, `searchView` has been initialized by `onCreateOptionsMenu` after `onStart`. 
So, I think, `onStart` is a bit earlier to call `resetSearchBox()`.

I tried to trigger `searchView` property initialization in `onPrepareOptionsMenu` by calling `invalidateOptionsMenu()`, it doesn't work. 
So, I updated this part [with different approach](https://github.com/rakutentech/android-miniapp/pull/111/commits/ecb5600475907648d7795bcddb2ea98378249afd).